src/clients/oauth.rs

```diff
@@ -992,6 +1000,27 @@ pub trait OAuthClient: BaseClient {
         Ok(())
     }
 
+    /// Resume a User’s Playback.
+    ///
+    /// Parameters:
+    /// - device_id - device target for playback
```

Comment:
1. The parameters list is missing the `position_ms` field. 
2. You should update the `CHANGELOG.md` since you have added a new endpoint.
3. According to the comment https://github.com/ramsayleung/rspotify/issues/231#issuecomment-881266383, it looks like the endpoint is missing the `offset` field

Reply:
> The parameters list is missing the position_ms field.

? The reference doesn't mention that. Did you add this comment on the wrong line of code?

Reply:
Oooh, I mean the `position_ms` is in the endpoint, but doesn't appear on parameter list comments above.

Reply:
There is one nitpick left, you should add `position_ms` in the comment like this:

```rust
    /// Parameters:
    /// - device_id - device target for playback
    /// - position_ms
```

Reply:
fixed